host/src/commands/client/agent.ts

```diff
@@ -33,6 +33,7 @@ export const handler = async (argv: Arguments<Options>): Promise<void> => {
     case 'unlock': await unlock(ad4mClient, passphrase); break;
     case 'status': await status(ad4mClient); break;
     case 'me': await me(ad4mClient); break;
+    case 'getApps': await getApps(ad4mClient); break;
```

Comment:
I consider the JS based CLI deprecated and superseded by the Rust based CLI (in the cli directory).

Reply:
Added the method to cli